### DISCUSSION TITLE
Scp like service

### DISCUSSION THREAD

**Discussion by buzzzo** [hoophq/hoop#549] [General]
Hi

It's easy with hoop.dev to create an ssh like session (using bash as command).
But what if i need to copy files to the remote service/machine ?

Teleport for example permits to copy files both via web interface both from it's cli.
Does hoop provide something similar ?

Thx

**Comment by buzzzo** [comment]
closed.

**Comment by Justin-Henrotte-Deuse** [comment]
Hi!

I see you've closed the issue. Is that because the feature now exists? I didn't see anything about it in the documentation.

Thanks

**Reply by buzzzo** [reply]
Hi

No, i've closed the issue because i've used another way to implement the scp like feature.
spoiler: used warpgate behind hoop as ssh reverse proxy.
